Testing/SystemTests/tests/analysis/DirectInelasticDiagnostic.py

```diff
@@ -48,7 +48,7 @@ def runTest(self):
 
         # Save the masked spectra nmubers to a simple ASCII file for comparison
         self.saved_diag_file = os.path.join(ms.config['defaultsave.directory'], 'CurrentDirectInelasticDiag.txt')
-        handle = file(self.saved_diag_file, 'w')
+        handle = open(self.saved_diag_file, 'w')
```

Comment:
Instead this should be
```python
with open(self.saved_diag_file, 'w') as handle:
```
then indent the code below and remove the explicit `close`